javascript/javascript1/week3/SmartPhoneUsageApplication.js

```diff
@@ -0,0 +1,56 @@
+//Adding an activity
+const activities = [];
+
+function addActivity(date , activity , duration)
+{
+    activities.push(
+        {
+            date :date , 
+            activity : activity , 
+            duration : duration
```

Comment:
You can use the shorthand here as well, because the properties share their names with the variables.